gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -107,10 +111,10 @@ struct SkipList {
             SkipListElement* twin = nullptr, bool is_vertex = false,
             std::pair<uintE, uintE> id = std::make_pair(UINT_E_MAX, UINT_E_MAX),
             double pb = 2, int num_dup = 2, size_t m = 10) {
-        rng.fork(index);
-        rng = rng.next();
-        auto rand_val = rng.rand() % UINT_E_MAX;
-        rng = rng.next();
+        //rng = rng.fork(0);
+        //rng = rng.next();
+        auto rand_val = rng[index]% UINT_E_MAX;
+        //rng = rng.next();
```

Comment:
```suggestion
        auto const rand_val = rng[index]% UINT_E_MAX;
```
You should delete lines that you do not want anymore rather than comment them out. This is all in version control; so, it is easy to go back and see what it used to be if one is curious.

Also note that this value doesn't change. Use `const` as much as possible.